src/status_im/common/keychain/events.cljs

```diff
@@ -176,6 +178,32 @@
                                   :key-uid key-uid
                                   :event   :keychain/password-hash-migration})))))))))
 
+(re-frame/reg-fx
+ :effects.keychain/save-keycard-keys
+ (fn [keycard-keys]
+   (let [{:keys [key-uid encryption-public-key whisper-private-key]}
+         (security/safe-unmask-data keycard-keys)]
+     (keychain/save-credentials
+      key-uid
+      key-uid
+      encryption-public-key
+      #(when-not %
+         (log/error
+          (str "Error while saving encryption-public-key"))))
```

Comment:
I think we know have a namespace for logging functions at `status-im.common.log`.
We can use the exported functions, re-frame events, or re-frame effects for logging.
* It's a little more verbose when using the functions, but it helpful to use the namespace in case we want to change our logging dependencies.

Reply:
this code is legacy i don't want to change it much, it should be refactored

Reply:
i'm not sure i understand what you mean here

Reply:
ah so atm the we're logging using this import and functions:
`[taoensso.timbre :as log]` and `log/error`

and now we have a namespace that wraps `taoensso.timbre` and provides a few ways to use our logging functions.
if you're curious about the namespace, you can take a peak here: https://github.com/status-im/status-mobile/blob/develop/src/status_im/common/log.cljs

---

my suggestion was that we could begin moving towards using the status namespace for logging. it's not a big priority, but it's a small thing we could refactor.

Reply:
thanks, still not sure what's the point, but done :)